crates/nu-command/src/conversions/into/glob.rs

```diff
@@ -0,0 +1,133 @@
+use nu_cmd_base::input_handler::{operate, CmdArgument};
+use nu_engine::CallExt;
+use nu_protocol::{
+    ast::{Call, CellPath},
+    engine::{Command, EngineState, Stack},
+    Category, Example, IntoPipelineData, PipelineData, ShellError, Signature, Span, SyntaxShape,
+    Type, Value,
+};
+
+struct Arguments {
+    cell_paths: Option<Vec<CellPath>>,
+}
+
+impl CmdArgument for Arguments {
+    fn take_cell_paths(&mut self) -> Option<Vec<CellPath>> {
+        self.cell_paths.take()
+    }
+}
+
+#[derive(Clone)]
+pub struct SubCommand;
+
+impl Command for SubCommand {
+    fn name(&self) -> &str {
+        "into glob"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("into glob")
+            .input_output_types(vec![
+                (Type::String, Type::Glob),
+                (
+                    Type::List(Box::new(Type::String)),
+                    Type::List(Box::new(Type::Glob)),
+                ),
+                (Type::Table(vec![]), Type::Table(vec![])),
+                (Type::Record(vec![]), Type::Record(vec![])),
+            ])
+            .allow_variants_without_examples(true) // https://github.com/nushell/nushell/issues/7032
+            .rest(
+                "rest",
+                SyntaxShape::CellPath,
+                "For a data structure input, convert data at the given cell paths.",
+            )
+            .category(Category::Conversions)
+    }
+
+    fn usage(&self) -> &str {
+        "Convert value to glob."
+    }
+
+    fn search_terms(&self) -> Vec<&str> {
+        vec!["convert", "text"]
+    }
+
+    fn run(
+        &self,
+        engine_state: &EngineState,
+        stack: &mut Stack,
+        call: &Call,
+        input: PipelineData,
+    ) -> Result<PipelineData, ShellError> {
+        glob_helper(engine_state, stack, call, input)
+    }
+
+    fn examples(&self) -> Vec<Example> {
+        vec![
+            Example {
+                description: "convert string to glob",
+                example: "'1234' | into glob",
+                result: Some(Value::test_string("1234")),
+            },
+            Example {
+                description: "convert filepath to string",
```

Comment:
I think this should say "to glob" instead of "to string", right?

Reply:
Yup.  It's change by #11992